packages/runtime/container-runtime/package.json

```diff
@@ -214,17 +214,6 @@
 		"typescript": "~5.1.6"
 	},
 	"typeValidation": {
-		"broken": {
-			"ClassDeclaration_ChannelCollection": {
-				"backCompat": false
-			},
-			"ClassDeclaration_ContainerRuntime": {
-				"backCompat": false,
-				"forwardCompat": false
-			},
-			"ClassDeclaration_DataStoreContexts": {
-				"forwardCompat": false
-			}
```

Comment:
@vladsud @agarwal-navin Do either of you know if these breaks are legitimate?

Reply:
I believe https://github.com/microsoft/FluidFramework/pull/20123 might be the ClassDeclaration_ChannelCollection breakage. If you try to actually build, and look at the the compile error it will tell you what the change is. (git blame on the disable will as well). Given that having extra suppressions here causes compile errors due to our use of @ts-expect-error I suspect there is a real break here.